dsa-solutions/gfg-solutions/0002-fascinating-number.md

```diff
@@ -0,0 +1,123 @@
+---
+id: fascinating-number
+title: Fascinating Number Problem (Geeks for Geeks)
+sidebar_label: 0002 - Fascinating Number 
+tags:
+  - Beginner
+  - String
+  - Find
+  - Multiplication
+  - Geeks for Geeks
+  - CPP
+  - Python
+  - DSA
+description: "This is a solution to the Fascinating Number problem on Geeks for Geeks."
+---
+
+## Problem Description
+
+Given a number N. Your task is to check whether it is fascinating or not.
+
+Fascinating Number: When a number(should contain 3 digits or more) is multiplied by 2 and 3, and when both these products are concatenated with the original number, then it results in all digits from 1 to 9 present exactly once.
+
+## Examples
+
+Example 1:
+
+Input: N = 192
```

Comment:
Bad practice:

```md
Input: N = 192
Output: Fascinating
Explanation: After multiplication with 2 and 3, and concatenating with original number, number will become 192384576 which contains all digits from 1 to 9.
```

Good Practice:

````md
```
Input: N = 192
Output: Fascinating
Explanation: After multiplication with 2 and 3, and concatenating with original number, number will become 192384576 which contains all digits from 1 to 9.
```
````